src/OrchardCore.Modules/OrchardCore.ContentFields/Views/TextField-IconPicker.Edit.cshtml

```diff
@@ -8,7 +8,8 @@
 
 <style name="fontpickerStyle" at="Head" asp-src="/OrchardCore.ContentFields/Styles/fontawesome-iconpicker-min.css" debug-src="/OrchardCore.ContentFields/Styles/fontawesome-iconpicker.css"></style>
 <style name="fontpickerCustomStyle" at="Head" asp-src="/OrchardCore.ContentFields/Styles/iconpicker-custom.css"></style>
-<script asp-name="bootstrap" at="Foot"></script>
+<script asp-name="popper" at="Foot"></script>
+<script asp-name="bootstrap" at="Foot" depends-on="popper"></script>
```

Comment:
@Skrypt says there is another bootstrap to use that has it

Reply:
@Skrypt Do you mean using asp-name="bootstrap-bundle" ?  It depends on popper.js.

Reply:
Yes that's what I meant.